layouts/partials/post_nav.html

```diff
@@ -0,0 +1,22 @@
+{{- if .Param "post_navigation" }}
```

Comment:
Delete this file

Reply:
Sorry about that. I must have missed that during the merge...